src/main/java/frc/RobotManager.java

```diff
@@ -72,6 +78,18 @@ public void robotPeriodic() {
 		AlertManager.reportAlerts();
 	}
 
+	private void createAutoReadyForConstructionChooser() {
```

Comment:
its true or false so
```suggestion
	private void createIsAutoReadyForConstructionChooser() {
```
on all the other ones as well